src/wallet/wallet.cpp

```diff
@@ -44,6 +44,7 @@ bool bSpendZeroConfChange = DEFAULT_SPEND_ZEROCONF_CHANGE;
 bool fWalletRbf = DEFAULT_WALLET_RBF;
 OutputType g_address_type = OUTPUT_TYPE_NONE;
 OutputType g_change_type = OUTPUT_TYPE_NONE;
+bool g_wallet_allow_fallback_fee = false; //<! will be defined via chainparams
```

Comment:
Following the same concept of #12408, move to wallet member?

Reply:
Fallback fee yes or now seems to me after a global setting (not per wallet).

Reply:
Ok then.

Reply:
In which case it could be a static member, just saying